src/components/Divider/Divider.tsx

```diff
@@ -29,16 +32,37 @@ const styleMap = {
 };
 
 export const Divider = ({
+    vertical = false,
     style = DividerStyle.Solid,
     height = DividerHeight.Small,
-    color: borderTopColor = '#CCC',
-}: DividerProps): ReactElement => (
-    <div className="tw-flex tw-items-center" style={{ height }} data-test-id="divider">
-        <hr
-            className={`tw-border-t tw-m-0 tw-w-full ${styleMap[style]}`}
-            style={{ borderTopColor }}
-            data-test-id="divider-hr"
-        />
-    </div>
-);
+    color = '#CCC',
+}: DividerProps): ReactElement => {
+    const verticalClassNames = `tw-w-px tw-h-full tw-border-r tw-m-0 ${styleMap[style]}`;
+    return (
+        <>
+            {vertical && (
+                <div
```

Comment:
I believe we're gonna need to use `aria-hidden="true"` on both cases, since `Divider` is a `div` with no content. It would be nice to confirm that with @LidijaStoilova before merging this...